projects/admin/src/app/record/custom-editor/circulation-settings/circulation-policy/circulation-policy.component.html

```diff
@@ -177,56 +188,102 @@ <h1 translate>Circulation Policy</h1>
-                formControlName="number_of_days_before_due_date">
-              <div
-                *ngIf="number_of_days_before_due_date.invalid && (number_of_days_before_due_date.dirty || number_of_days_before_due_date.touched)"
-                class="text-danger pt-1">
-                <div *ngIf="number_of_days_before_due_date.errors.required" translate>
-                  Days before due date is required.
-                </div>
-                <div *ngIf="number_of_days_before_due_date.errors.min" translate>
-                  Days before due date must be greater or equal to 0.
+              <div class='form-group'>
+                <div class="custom-control custom-switch">
+                  <input
+                    formControlName="send_first_reminder"
+                    class="custom-control-input"
+                    type="checkbox"
+                    id="send_first_reminder"
+                    [checked]="send_first_reminder.value">
+                  <label class="custom-control-label" for="send_first_reminder" translate>
+                    Send a first reminder
+                  </label>
                 </div>
               </div>
             </div>
-          </div>
-
-          <div class="form-group row" *ngIf="allow_checkout.value">
-            <label for="overdueAmount" class="col-sm-2 col-form-label required">{{ 'Fees' | translate }}
-              ({{ currency }})</label>
-            <div class="col-sm-4">
-              <input type="number" class="form-control" id="overdueAmount" formControlName="reminder_fee_amount">
-              <div *ngIf="reminder_fee_amount.invalid && (reminder_fee_amount.dirty || reminder_fee_amount.touched)"
-                class="text-danger pt-1">
-                <div *ngIf="reminder_fee_amount.errors.required" translate>
-                  Overdue amount is required.
+            <div class="col-sm-8 border rounded pt-3" *ngIf="send_first_reminder.value">
+              <div class="form-group row">
+                <label class="col-sm-6 col-form-label text-right" for="daysAfterControl" translate>
+                  Sending how many days after the due date
+                </label>
+                <input
+                  type="number"
+                  class="form-control required col-sm-3"
+                  id="daysAfterControl"
+                  formControlName="number_of_days_after_due_date">
+                <div
+                  *ngIf="number_of_days_after_due_date.invalid && (number_of_days_after_due_date.dirty || number_of_days_after_due_date.touched)"
+                  class="text-danger pt-1">
+                  <div *ngIf="number_of_days_after_due_date.errors.required" translate>
+                    First reminder days is required.
+                  </div>
+                  <div *ngIf="number_of_days_after_due_date.errors.min" translate>
+                    First reminder days must be greater than 0.
```

Comment:
This one too, I think it could be improved and checked with @pronguen 

My proposition:
```suggestion
                    The number of days to wait after the due date before sending the first reminder must be greater than 0.
```